### DISCUSSION TITLE
Implement credential api

### DISCUSSION THREAD

**Discussion by carakas** [forkcms/forkcms#3345] [Ideas / Feature Requests]
## Type

- Enhancement

## Problem description

people forget passwords

this helps them, currently only chrome, but a lot of people use it so it is worth it.

Netflix had a 20% drop in support questions of forgotten credentials after implementing this

**Comment by tijsverkoyen** [comment]
I think you mean https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API? If so, this is a experimental feature, so I won't bother at this moment

**Comment by carakas** [comment]
it is already fully implemented in chrome and in android, so I think it is something they will push through, if companies like netflix (was the big example I remember) I think it's wurth looking in to, especially since it's not that much work
http://caniuse.com/#feat=credential-management

**Comment by jessedobbelaere** [comment]
I didn't attend this talk but looks nice to have indeed 👍 
https://docs.google.com/presentation/d/1_X_YibtMss9-MfYO0BFI1798vW0vQec4p_j8VPi4-xY/preview?slide=id.p

**Comment by jeroendesloovere** [comment]
I don't think such a feature is something we need to add to Fork CMS.
Since my browser (safari) is already auto-completing username and password.

**Comment by carakas** [comment]
It is more than autocompleting :)

**Comment by carakas** [comment]
It is supported now by most browsers so it looks like it is still a nice feature
https://caniuse.com/?search=credentials
